docs/advanced/security.rst

```diff
@@ -0,0 +1,52 @@
+=================================
+Cluster Security & Authentication
+=================================
+
+A Flocker cluster comprises a control service and convergence agents, along with some command line tools that are provided to interact with and manage the cluster (see :doc:`architecture`).
+
+Flocker uses `Transport Layer Security <http://en.wikipedia.org/wiki/Transport_Layer_Security>`_ (TLS) to authenticate components of a cluster against each other, using a technique known as client certification.
+
+This ensures that the control service, convergence agents and API end users can be sure that they are communicating with a verified component of a cluster, helping to prevent unauthorised access and mitigating some potential attack vectors.
+
+Client Certification Overview
+=============================
+
+When you :doc:`install the flocker-node package <../indepth/installation>`, you are asked to run the ``flocker-ca`` tool provided as part of ``flocker-cli`` to generate a certificate authority for your cluster, as well as certificates and private keys for the control service and convergence agents.
+
+A certificate authority, also known as a root certificate, is a special type of TLS certificate that is authorised to generate other certificates by signing another key's `certificate signing request <http://en.wikipedia.org/wiki/Certificate_signing_request>`_ with its own private key.
+
+This provides a means to verify the identity of an entity (for example, the control service or an API end user) that presents itself as being an authorised component of a cluster.
```

Comment:
A CSR does not provide any means to verify. The verification is a separate Out-of-band operation. Why are we even explaining all this? Can't we just refer to an external link, and say "Here's what you need to do to make Flocker work"